extensions/common/json_util.cc

```diff
@@ -20,60 +20,91 @@
 namespace Wasm {
 namespace Common {
 
-::nlohmann::json JsonParse(absl::string_view str) {
-  return ::nlohmann::json::parse(str, nullptr, false);
+absl::optional<JsonObject> JsonParse(absl::string_view str) {
+  const auto result = JsonObject::parse(str, nullptr, false);
+  if (result.empty() || result.is_discarded() || !result.is_object()) {
```

Comment:
Is empty an invalid case?

Reply:
Sure. I think that we shouldn't treat empty as a valid value because we can't handle discarded results which are considered as parse error won't be processed when empty value.

Reply:
I think that there is no problem to treat empty values as parse failed, but It may cause confusion in the future. I changed this as not valid value.